webapp/src/main/java/com/box/l10n/mojito/service/ai/RepositoryLocaleAIPromptRepository.java

```diff
@@ -0,0 +1,30 @@
+package com.box.l10n.mojito.service.ai;
+
+import com.box.l10n.mojito.entity.RepositoryLocaleAIPrompt;
+import java.util.List;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+public interface RepositoryLocaleAIPromptRepository
+    extends JpaRepository<RepositoryLocaleAIPrompt, Long> {
+
+  @Query(
+      "SELECT rlap.repository.id FROM RepositoryLocaleAIPrompt rlap WHERE rlap.aiPrompt.id = :aiPromptId")
+  List<Long> findRepositoryIdsByAiPromptId(@Param("aiPromptId") Long aiPromptId);
+
+  @Query(
```

Comment:
Maybe I am misunderstanding something but these method names are similar yet their where clauses are different `rlap.disabled = false` and `aip.deleted = false` is the difference. 

This means to me that either the names are deceptive (they are different things) or there is a mistake when copying the queries or perhaps I am missing some nomenclature / taxonomy information

Reply:
Need to add `rlap.disabled` to the second query

Reply:
Correction on this, I need both disabled/enabled in the query to handle the use case of disabling a specific locale for AI translation e.g. I have AI translation enabled for a mojito repository but I want to specifically disable it for French for some reason